oak-segment-tar/src/test/java/org/apache/jackrabbit/oak/segment/SegmentBlobTrackerRegistrationTest.java

```diff
@@ -19,36 +19,51 @@
 
 package org.apache.jackrabbit.oak.segment;
 
-import static org.apache.jackrabbit.guava.common.collect.Maps.newHashMap;
 import static org.apache.sling.testing.mock.osgi.MockOsgi.deactivate;
 
-import java.util.Map;
+import java.util.Collection;
+import java.util.Dictionary;
+import java.util.Hashtable;
 
 import org.apache.jackrabbit.oak.plugins.blob.AbstractBlobTrackerRegistrationTest;
 import org.apache.jackrabbit.oak.plugins.blob.BlobTrackingStore;
+
+import org.apache.sling.testing.mock.osgi.MockOsgi;
 import org.osgi.framework.InvalidSyntaxException;
 import org.osgi.framework.ServiceReference;
 
+import static org.junit.Assert.assertNotNull;
+
 /**
  * Tests OSGi registration for {@link BlobTrackingStore} in {@link SegmentNodeStoreService}.
  */
 public class SegmentBlobTrackerRegistrationTest extends AbstractBlobTrackerRegistrationTest {
 
-    private SegmentNodeStoreService service;
-
     @Override
     protected void registerNodeStoreService() {
-        Map<String, Object> properties = newHashMap();
+        Hashtable<String, Object> properties = new Hashtable<>();
         properties.put(SegmentNodeStoreService.CUSTOM_BLOB_STORE, true);
         properties.put(SegmentNodeStoreService.REPOSITORY_HOME_DIRECTORY, repoHome);
-        service = context.registerInjectActivateService(new SegmentNodeStoreService(), properties);
+        SegmentNodeStoreService service = new SegmentNodeStoreService();
+        
```

Comment:
@kwin Would you take a look at the following lines? I wonder if MockBundleContext introduced a bug in the new version or if I'm missing something.